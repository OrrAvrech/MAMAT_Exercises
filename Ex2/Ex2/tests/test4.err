Insert execution failed.
Add execution failed.
Change failed: not enough parameters.
Remove execution failed.
