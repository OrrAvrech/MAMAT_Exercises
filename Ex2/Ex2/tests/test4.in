Print
Insert 100001 D
Insert 100000 D
Add 100001 D AAA R
Add 100000 D AAA R
Add 1 D ABC E
Add 2 D AAB E
Add 3 D ACB E
Add 4 D AHA R
Print
Delay AAB
Print
Insert 2 I
Add 19 I APA E
Delay APA
Change APA KFC
Print
Change KFC ABI
Print
Delay ABI
Print
Change ABC ACB
Delay ACB
Print
Change AAA
Change AAA BBB
Add 20 I BBB E
Add 22 D PPP E
Print
Remove 1
Print
Add 919 D ACB R
Add 900 D QIW R
Print
Delay ACB
Print