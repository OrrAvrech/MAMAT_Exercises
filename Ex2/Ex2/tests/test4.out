Airport status:

Airport status:
Runway 100000 domestic
5 flights are waiting:
Flight 1 D ABC E
Flight 2 D AAB E
Flight 3 D ACB E
Flight 100000 D AAA R
Flight 4 D AHA R

Airport status:
Runway 100000 domestic
5 flights are waiting:
Flight 1 D ABC E
Flight 3 D ACB E
Flight 2 D AAB E
Flight 100000 D AAA R
Flight 4 D AHA R

Airport status:
Runway 100000 domestic
5 flights are waiting:
Flight 1 D ABC E
Flight 3 D ACB E
Flight 2 D AAB E
Flight 100000 D AAA R
Flight 4 D AHA R
Runway 2 international
1 flights are waiting:
Flight 19 I KFC E

Airport status:
Runway 100000 domestic
5 flights are waiting:
Flight 1 D ABC E
Flight 3 D ACB E
Flight 2 D AAB E
Flight 100000 D AAA R
Flight 4 D AHA R
Runway 2 international
1 flights are waiting:
Flight 19 I ABI E

Airport status:
Runway 100000 domestic
5 flights are waiting:
Flight 1 D ABC E
Flight 3 D ACB E
Flight 2 D AAB E
Flight 100000 D AAA R
Flight 4 D AHA R
Runway 2 international
1 flights are waiting:
Flight 19 I ABI E

Airport status:
Runway 100000 domestic
5 flights are waiting:
Flight 2 D AAB E
Flight 1 D ACB E
Flight 3 D ACB E
Flight 100000 D AAA R
Flight 4 D AHA R
Runway 2 international
1 flights are waiting:
Flight 19 I ABI E

Airport status:
Runway 100000 domestic
6 flights are waiting:
Flight 2 D AAB E
Flight 1 D ACB E
Flight 3 D ACB E
Flight 22 D PPP E
Flight 100000 D BBB R
Flight 4 D AHA R
Runway 2 international
2 flights are waiting:
Flight 19 I ABI E
Flight 20 I BBB E

Airport status:
Runway 100000 domestic
6 flights are waiting:
Flight 2 D AAB E
Flight 1 D ACB E
Flight 3 D ACB E
Flight 22 D PPP E
Flight 100000 D BBB R
Flight 4 D AHA R
Runway 2 international
2 flights are waiting:
Flight 19 I ABI E
Flight 20 I BBB E

Airport status:
Runway 100000 domestic
8 flights are waiting:
Flight 2 D AAB E
Flight 1 D ACB E
Flight 3 D ACB E
Flight 22 D PPP E
Flight 100000 D BBB R
Flight 4 D AHA R
Flight 919 D ACB R
Flight 900 D QIW R
Runway 2 international
2 flights are waiting:
Flight 19 I ABI E
Flight 20 I BBB E

Airport status:
Runway 100000 domestic
8 flights are waiting:
Flight 2 D AAB E
Flight 22 D PPP E
Flight 1 D ACB E
Flight 3 D ACB E
Flight 100000 D BBB R
Flight 4 D AHA R
Flight 900 D QIW R
Flight 919 D ACB R
Runway 2 international
2 flights are waiting:
Flight 19 I ABI E
Flight 20 I BBB E

